pkg/host/internal/network/network.go

```diff
@@ -290,3 +293,43 @@ func (n *network) SetDevlinkDeviceParam(pciAddr, paramName, value string) error
 	}
 	return nil
 }
+
+// EnableHwTcOffload makes sure that hw-tc-offload feature is enabled if device supports it
+func (n *network) EnableHwTcOffload(ifaceName string) error {
+	log.Log.V(2).Info("EnableHwTcOffload(): enable offloading", "device", ifaceName)
+	featureName := "hw-tc-offload"
```

Comment:
it's used only once here and in tests so I'm OK to have it here instead of constants

Reply:
yes, I think there is no need to put the feature name to a const